src/promptflow/promptflow/tracing/contracts/trace.py

```diff
@@ -16,6 +16,7 @@ class TraceType(str, Enum):
     LANGCHAIN = "LangChain"
     FLOW = "Flow"
     EMBEDDING = "Embedding"
+    FEEDBACK = "Feedback"
```

Comment:
Is it good to add a new "Feedback" type here? Is it too specific that only used by one scenario? cc @thy09

Reply:
This is optional for serving scenario, I can remove this if there's concern.

Reply:
Let me remove it first,  we can add it back if there's requirement for this.